tests/ui/structs/default-field-values/const-trait-default-field-value.rs

```diff
@@ -0,0 +1,28 @@
+//@ check-pass
+
+// Ensure that `default_field_values` and `const_default` interact properly.
+
+#![feature(default_field_values)]
+#![feature(const_trait_impl)]
```

Comment:
Do you think it would be worthwhile to also test default field values (set to a `Default::default()` like here) where the struct init is at comptime (e.g., using `const { ... }`) (it could utilize `#[derive_const]`) or is this already covered somewhere else?